ecc/Makefile

```diff
@@ -1,11 +1,13 @@
+# Copyright 2019 Intel Corporation
 # Copyright IBM Corp. All Rights Reserved.
 #
 # SPDX-License-Identifier: Apache-2.0
 
 TOP = ..
 include $(TOP)/build.mk
 
-LD_LIBRARY_PATH := LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$$(pwd)/enclave/lib
+LD_LIBRARY_PATH := LD_LIBRARY_PATH=${LD_LIBRARY_PATH}
+
```

Comment:
I think you rely on ENCLAVE_SO_PAT for finding fpc-c chaincode (enclave.so).  Maybe also add a define here so all the parameter for the docker build would be at a single place?  (we might also add a commenting header saying that these are the docker build params?)

Reply:
not sure what you mean, but I created an internal variable, now check the value, and clarified what the docker build parameters are

Reply:
My comment was in the context of the three definitions CC_NAME, NET_ID and PEER_ID which are following at the location i commented:  These three variables define almost but not quite the "api" you would use for 'docker-fpc-app'.  The almost being that you also will have to define that path to the code which i thought is ENCLAVE_SO_PATH.   Hence my suggestion to also add a ?= define or alike for ENCLAVE_SO_PATH to the other three defines and add a comment above the four of them that this defines the 'make docker-fpc-app' api